test/src/main/java/org/axonframework/test/fixture/AxonTestPhase.java

```diff
@@ -186,7 +185,7 @@ default Given event(@Nonnull Object payload, @Nonnull Map<String, ?> metaData) {
          * @param events The lists of events to publish.
          * @return The current Given instance, for fluent interfacing.
          */
-        Given events(@Nonnull List<?>... events);
+        Given events(@Nonnull List<?> events);
```

Comment:
It would be nice to overload this method to be able to have `Object... events`

Reply:
Added, also for commands